Casks/rustdesk.rb

```diff
@@ -0,0 +1,24 @@
+cask "rustdesk" do
+  version "1.1.9"
+  sha256 "3b9662a8d5d8a6cc994f6edee132f116f35dee4a8a17c3612b784376f7da931f"
+
+  url "https://github.com/rustdesk/rustdesk/releases/download/#{version}/rustdesk-#{version}.dmg",
+      verified: "github.com/rustdesk/rustdesk/"
+  name "RustDesk"
+  desc "Open source virtual / remote desktop infrastructure for everyone"
```

Comment:
```suggestion
  desc "Open source virtual/remote desktop application"
```

[Do not use empty marketing jargon.](https://docs.brew.sh/Cask-Cookbook#stanza-desc)